docs/content/preview/drivers-orms/go/pq.md

```diff
@@ -52,7 +50,7 @@ import (
 )
 ```
 
-### Step 2: Connect to YugabyteDB database
+### Step 2: Set up the database connection
 
 Go applications can connect to YugabyteDB using the `sql.Open()` function. The `sql` package includes all the functions or structs required for working with YugabyteDB.
 
```

Comment:
the URL/connection string isn't Go. Maybe text or sh

Same comments RE SSL table